### PR TITLE
Updated Git documentation.

### PR DESCRIPTION
**Pull request by MBudreviciusBentley**
## Changes

Updated packages under `ui` directory `README.md` and `package.json`files.

## Testing

N/A

### CODE REVIEW

File: ui/imodel-components-react/README.md [hunk 1]
```diff
@@ -6,6 +6,19 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
...
 
-## Documentation
+## Usage
+
+To get started with React components from the `@itwin/imodel-components-react` package, simply import the necessary components and utilities:
+
+```tsx
+import { BaseSolarDataProvider } from "@itwin/imodel-components-react";
+
+export const Basic: Story = {
```
Comment (GerardasB) [inline ui/imodel-components-react/README.md:16]: Let's avoid storybook specific snippets.

File: ui/components-react/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@itwin/components-react",
   "version": "4.17.0-dev.0",
-  "description": "iTwin.js UI complex components",
+  "description": "A react component library for iTwin.js UI data-oriented components",
```
Comment (GerardasB) [inline ui/components-react/package.json:4]:
```suggestion
  "description": "A react component library of iTwin.js UI data-oriented components",
```

**Review by GerardasB — CHANGES_REQUESTED** [review]

File: ui/core-react/README.md [hunk 3]
```diff
@@ -6,6 +6,29 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
...
+    return (
+      <ContextMenu {...props}>
+        <ContextSubMenu label="Label 1" id="1">
+          <ContextMenuItem>Label 1</ContextMenuItem>
+        </ContextSubMenu>
+      </ContextMenu>
+    );
+  },
+};
+
```
[the remark this answers is not in the document]
Comment (GerardasB) [reply ui/core-react/README.md:31]: I think we might need a separate setup and usage sections for each package..

[the remark this answers is not in the document]
Comment (MBudreviciusBentley) [reply ui/core-react/README.md]: Updated the code sample.

File: ui/appui-react/README.md [hunk 4]
```diff
@@ -16,6 +16,29 @@ actions using reducers.
...
+    "item1",
+    StatusBarSection.Left,
+    10,
+    <MessageCenterField />
+  );
+
+  return {
+    custom1
+  };
+}
```
[the remark this answers is not in the document]
Comment (MBudreviciusBentley) [reply ui/appui-react/README.md]: Updated the code sample.

File: ui/components-react/README.md [hunk 5]
```diff
@@ -6,6 +6,40 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
...
+  return {
+    operator: PropertyFilterRuleGroupOperator.Or,
+    rules: [
+      {
+        property: property,
+        operator: "like",
+      },
+    ],
+  };
+}
```
[the remark this answers is not in the document]
Comment (MBudreviciusBentley) [reply ui/components-react/README.md]: Updated the code sample.

File: ui/imodel-components-react/README.md (hunk 1 above)
Comment (MBudreviciusBentley) [reply ui/imodel-components-react/README.md]: Updated the code sample.

File: common/changes/@itwin/appui-react/MBudrevicius-UpdateDocumentation_2024-09-30-12-32.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "Updated documentation",
```
Comment (GerardasB) [inline common/changes/@itwin/appui-react/MBudrevicius-UpdateDocumentation_2024-09-30-12-32.json:5]: I'd leave these empty, README update is a somewhat internal change, consumers don't need to care about that when upgrading
```suggestion
      "comment": "",
```

File: ui/components-react/README.md [hunk 7]
```diff
@@ -6,6 +6,21 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
...
+## Usage
 
-See the [iTwin.js](https://www.itwinjs.org/learning/ui/components/) documentation for more information.
+To get started with the `@itwin/components-react` package, simply import the necessary components and utilities:
+
+```tsx
+import * as React from "react";
+import { Panel, Title } from "@itwin/components-react";
+
+function getListPanel(): React.ReactNode {
```
Comment (GerardasB) [inline ui/components-react/README.md:17]: Keep implicit types. Component names are capitalized.
```suggestion
function ListPanel() {
```

File: ui/imodel-components-react/README.md [hunk 8]
```diff
@@ -6,6 +6,22 @@ Copyright © Bentley Systems, Incorporated. All rights reserved. See LICENSE.md
...
+
+```tsx
+import * as React from "react";
+import { TimelineComponent } from "@itwin/imodel-components-react";
+
+export function GetTimelineComponent() {
+  return (
+    <TimelineComponent
+      startDate={new Date(1, 1, 1)}
+      endDate={new Date(2, 2, 2)}
```
Comment (GerardasB) [inline ui/imodel-components-react/README.md:21]: Dates are [tricky](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#year), pick something more reasonable:
```suggestion
      startDate={new Date(2024, 8, 30)}
      endDate={new Date()}
```

**Review by GerardasB — APPROVED** [review]

**Pull request merged**